src/components/Confirmation/SubRow/SubRow.styles.js

```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/core';
+import { Color } from 'lib/theme';
+
+export const styles = {
+  label: css({
+    color: Color.GRAY,
+    fontWeight: 500,
```

Comment:
do we need this font weight? can we pass `type` to the Text component instead?

Reply:
✂️